examples/segmenter/segmenter.go

```diff
@@ -109,7 +108,8 @@ func (s *Segmenter) MakeMuxedInitSegment() (*mp4.InitSegment, error) {
 	init.Moov.Mvex.AddChild(&mp4.MehdBox{FragmentDuration: int64(inMovieDuration)})
 	for _, tr := range s.tracks {
 		init.AddEmptyTrack(tr.timeScale, tr.trackType, tr.lang)
-		outTrak := init.Moov.Traks[len(init.Moov.Traks)-1]
+		// outTrak := init.Moov.Traks[len(init.Moov.Traks)-1]
```

Comment:
The init.EmptyTrack() call above should be removed, since otherwise you create two tracks.